docs/AboutUs.md

```diff
@@ -24,7 +24,7 @@ You can reach us at the email `e0406158@u.nus.edu`
 <img src="images/marcustw.png" width="200px">
 
 [[github](http://github.com/marcustw)]
-[[portfolio](team/johndoe.md)]
+[[portfolio](team/marcustan.md)]
```

Comment:
Should this be in the form of team/githbub_username_in_lower_case.md? i.e. marcustw

Reply:
Thank you for pointing it out! I've made the necessary changes.